### PR TITLE
add oit links

### PR DESCRIPTION
**Pull request by paulstey**
This PR adds a few links to OIT main page; this is to address comments from OIT directors. This PR also includes a bit of content update to the lead on the `About` page.

#### PR Summary
Briefly describe the changes.

#### Check the types of changes present in this PR:
- [ ] :bug: Bug
- [ ] :dragon: Feature
- [ ] :frog: Data (data folder - people/opportunities)
- [x] :dog: Content (content folder)
- [ ] :blowfish: Other. Specify:

#### Checklist:
- [x] Commitizen was used for all commits.
- [x] Local site looks as expected after changes.
- [x] Contributing guidelines were followed.
- [x] If changes affect development, was the README updated?

##### If PR to `production`
- [ ] Development site (https://datasci.brown.edu) was checked and looks as expected.

### CODE REVIEW

File: content/english/about/_index.md [hunk 1]
```diff
@@ -17,9 +17,11 @@ items:
   icon: comments
 
 mission:
-  lead: 'The Center for Computation and Visualization provides high-performance computing
-    and visualization services to the Brown community. We also collaborate with researchers
-    on projects across vast range of disciplines. '
+  lead: 'The Center for Computation and Visualization (CCV) is a division of the [Office of
+    Information technology (OIT)](https://it.brown.edu). CCV provides research computing
```
Comment (mcmcgrath13) [inline content/english/about/_index.md]:
```suggestion
    Information Technology (OIT)](https://it.brown.edu). CCV provides research computing 
```

File: content/english/services/_index.md [hunk 2]
```diff
@@ -5,12 +5,10 @@ tagTitle: Services - Center for Computation and Visualization
 tagDescription: Learn more about the CCV's high-performance computing and visualization services. 
 
 hero:
-  lead: 'CCV provides computing infrastructure, consulting, and support to the Brown
-    Community. We have HPC specialists, data scientists, and research software engineers
-    available to work with researchers. We frequently partner with researchers on
-    projects that may span weeks, months, or years. These partnerships can in some
-    cases involve a researcher using grant funds to support one of our data scientists
-    or research software engineers. '
+  lead: 'CCV provides computing infrastructure, consulting, and support to the Brown Community. We
```
Comment (mcmcgrath13) [inline content/english/services/_index.md:8]: what happened to the formatting here? did any content change?

**Review by mcmcgrath13 — CHANGES_REQUESTED** [review]

File: content/english/about/_index.md (hunk 1 above)
Comment (paulstey) [reply content/english/about/_index.md]: Good catch! Thank you, Mary!

File: content/english/services/_index.md (hunk 2 above)
Comment (paulstey) [reply content/english/services/_index.md:8]: Good question! I'm not sure, I think I tweaked some indents. There is no content change.

**Review by mcmcgrath13 — APPROVED** [review]

**Review by singhsaluja — APPROVED** [review]

**Pull request merged**